### PR TITLE
update ftr information box with recommendation playback and add integ…

### PR DESCRIPTION
**Pull request by xebu**
…ration testing with firefox

### CODE REVIEW

File: server/views/partials/recommendation/recallTypeFTR.njk [hunk 1]
```diff
@@ -4,12 +4,51 @@
...
         <li>the MAPPA level is 2 or 3 at the point of recall</li>
     </ul>
     This applies to people aged 18 and over only.
 </p>
+<p>
+    You have said that:
+    <ul>
+        <li>{{ recommendation.personOnProbation.name }}
+            <strong>is</strong>
+            {% if recommendation.isUnder18 == false or recommendation.isUnder18 == 'NO' %}
```
Comment (DarrenW-HMPPS) [inline server/views/partials/recommendation/recallTypeFTR.njk:19]: Have we not got this typed?  Surely it comes from the API/Database as boolean.

File: server/views/partials/recommendation/recallTypeFTR.njk [hunk 2]
```diff
@@ -4,12 +4,51 @@
...
+            {% endif %}
+            above 1</li>
+        <li>{{ recommendation.personOnProbation.name }}
+            <strong>has</strong>
+            {% if recommendation.hasBeenConvictedOfSeriousOffence == false or recommendation.hasBeenConvictedOfSeriousOffence == 'NO' %}
+                <strong>not</strong>
+            {% endif %}
+            been charged with a serious offence.</li>
+    </ul>
+    {% if recommendation.isUnder18 or recommendation.isSentence12MonthsOrOver or isMappaLevelAbove1 or hasBeenConvictedOfSeriousOffence %}
```
Comment (DarrenW-HMPPS) [inline server/views/partials/recommendation/recallTypeFTR.njk:42]: Mismatch here with `recommendation.` not being included on the last 2 conditions.

There is an assumption here that these values are boolean, whereas the code above is checking for 'NO'

**Review by DarrenW-HMPPS — CHANGES_REQUESTED** [review]

File: server/views/partials/recommendation/recallTypeFTR.njk (hunk 1 above)
Comment (robinandrewdevilliers) [reply server/views/partials/recommendation/recallTypeFTR.njk:19]: isUnder18?: boolean,
  isMappaLevelAbove1?: boolean,
  isSentence12MonthsOrOver?: boolean,
  hasBeenConvictedOfSeriousOffence?: boolean,

Comment (xebu) [reply server/views/partials/recommendation/recallTypeFTR.njk:19]: corrected - I did double check after @DarrenW-HMPPS 's comment. Thanks @robinandrewdevilliers .

File: server/views/partials/recommendation/recallTypeFTR.njk (hunk 2 above)
Comment (xebu) [reply server/views/partials/recommendation/recallTypeFTR.njk:42]: corrected

**Review by DarrenW-HMPPS — APPROVED** [review]

**Pull request merged**